maskrcnn_benchmark/data/datasets/abstract_coco.py

```diff
@@ -0,0 +1,41 @@
+# Testing unit for PR #860
+# https://github.com/facebookresearch/maskrcnn-benchmark/pull/860
```

Comment:
This is a completely auxiliary test module, to show that COCO can be wrapped into an Abstract dataset and can be evaluated with the COCO tools.